Formula/geant4.rb

```diff
@@ -0,0 +1,100 @@
+class Geant4 < Formula
+  desc "Simulation toolkit for particle transport through matter"
+  homepage "http://geant4.cern.ch"
+  url "http://geant4.cern.ch/support/source/geant4.10.04.p02.tar.gz"
```

Comment:
Here's the secure alternative for the above URL:
https://geant4-data.web.cern.ch/geant4-data/releases/source/geant4.10.04.p02.tar.gz

Reply:
Thanks!